### PR TITLE
[SELS-TASK] User can view (other's or self) profile details such as name and avatar.

### PR DESCRIPTION
**Pull request by ejanton-sun**
# Asana Link
https://app.asana.com/0/1200129106185752/1200150975082479/f

# Commands
none.

# Pre-conditions
none.

# Expected Output
User can view other's profile and can follow or unfollow.

# Notes
Users cannot follow that is already followed user.

### CODE REVIEW

File: resources/views/user/profile.blade.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                </div>
+                <div class="col">
+                    <p>{{ $user->following() }}</p>
+                    Following
+                </div>
+                @if ($user->id != Auth::id())
+                <form action="{{route(!in_array($user->id,$followUser)?"user.follow":"user.unfollow",$user)}}" method="post">
+                    @csrf
+                    <input class="btn" type="submit" value="{{!in_array($user->id,$followUser)?"Follow":"unfollow"}}">
+                </form>
```
Comment (RunstyleX) [inline resources/views/user/profile.blade.php]:
```suggestion
                <form action="{{route(!in_array($user->id,$followUser)?"user.follow":"user.unfollow",$user)}}" method="post">
                        @csrf
                        <input class="btn" type="submit" value="{{!in_array($user->id,$followUser)?"Follow":"unfollow"}}">
                </form>
```

File: resources/views/user/profile.blade.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-2 ">
+            <img src="{{ url('/storage/uploads/avatars/'.$user->avatar) }}" class="card-img-top"
+                alt="{{$user->avatar}}">
+            <hr>
+            <div class="row text-center">
+                <div class="col">
+                    <p> {{ $user->followers()}}
+                    </p>
```
Comment (RunstyleX) [inline resources/views/user/profile.blade.php]:
```suggestion
                    <p> {{ $user->followers()}}</p>
```

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -69,6 +69,11 @@ class="navbar-toggler"
                         </li>
                         @endif
                         @else
+                        @auth
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{route('user.show',Auth::user())}}"> {{ Auth::user()->name }}</a>
+                        </li>
```
Comment (RunstyleX) [inline resources/views/layouts/app.blade.php]: Indent

File: resources/views/layouts/app.blade.php [hunk 4]
```diff
@@ -69,6 +69,11 @@ class="navbar-toggler"
                         </li>
                         @endif
                         @else
```
Comment (RunstyleX) [inline resources/views/layouts/app.blade.php:71]: Remove?

**Review by RunstyleX — CHANGES_REQUESTED** [review]

File: resources/views/layouts/app.blade.php (hunk 4 above)
Comment (ejanton-sun) [reply resources/views/layouts/app.blade.php:71]: No. Because the Nav link will be destroyed and it is part of  @guest blade directives.

**Pull request merged**